pyaedt/icepak.py

```diff
@@ -3649,7 +3652,13 @@ def _parse_variation_data(self, quantity, variation_type, variation_value, funct
                 variation_value = [variation_value[0], "pwl({},Temp)".format(variation_value[1])]
             out_dict["Variation Value"] = "[{}]".format(", ".join(['"' + str(i) + '"' for i in variation_value]))
         else:
-            out_dict["Variation Value"] = "[{}]".format(", ".join([str(i) for i in variation_value]))
+            try:
+                out_dict["Variation Value"] = "[{}]".format(", ".join([str(i) for i in variation_value]))
```

Comment:
@lorenzovecchietti I would suggest checking the variation_value type, we prefer to use pyaedt error handling instead of using try/except.